### PR TITLE
Updated default value for the acrylic setting

### PR DESCRIPTION
**Pull request by yair100**

### CODE REVIEW

File: Files/ViewModels/SettingsViewModel.cs [hunk 1]
```diff
@@ -515,7 +515,7 @@ public bool PinRecycleBinToSideBar
         /// </summary>
         public bool IsAcrylicDisabled
```
Comment (d2dyno1) [inline Files/ViewModels/SettingsViewModel.cs:516]: That's a weird setting name - `IsAcrylicDisabled`. Wouldn't it be better if it was called `IsAcrylicEnabled`?

**Review by gave92 — APPROVED** [review]

File: Files/ViewModels/SettingsViewModel.cs (hunk 1 above)
Comment (yaichenbaum) [reply Files/ViewModels/SettingsViewModel.cs:516]: @d2dyno1 This was added before we started using converters, we can definitely change around the name. Do you want to work on this?

**Comment by gave92** [comment]
Sort of unrelated: perhaps the setting to show the library section should also be enable by default? Users who update will likely think they lost those pinned folders and go to add those back manually ;)

File: Files/ViewModels/SettingsViewModel.cs (hunk 1 above)
Comment (d2dyno1) [reply Files/ViewModels/SettingsViewModel.cs:516]: @yair100 sure!

**Comment by yaichenbaum** [comment]
@gave92 We can show the library section by default if that's what users are looking for, but users who updated from previous versions will still have the library items pinned to the favorites section.

**Comment by gave92** [comment]
@yair100 it's all right then. I haven't noticed because when I tested that PR I lost all my pinned folders 😂

**Comment by yaichenbaum** [comment]
@gave92 That's been resolved, I changed the property name we stored in the json file by mistake.

**Pull request merged**